bundle-workflow/src/run_integ_test.py

```diff
@@ -100,10 +104,12 @@ def sync_dependencies_to_maven_local(work_dir, manifest_build_ver):
 
 def main():
     args = parse_arguments()
+    logging.getLogger('').handlers = []
```

Comment:
This shouldn't be needed.

Reply:
Yeah the handlers were not getting cleared on my local. will remove it from here

Reply:
Done